UselessMine.cpp

```diff
@@ -227,16 +221,28 @@ void UselessMine::Init (const char* commandLine)
 
     // Save the location of the file so we can reload after
     deathMessagesFile = commandLine;
-
+    defusalMessagesFile = commandLine;
```

Comment:
Since we'll be having two separate files now, we'll have to tokenize the `commandLine` variable so the following load command will work:

```
-loadplugin UselessMine.so,UselessMine.deathMessages,UselessMine.defuseMessages
```

You can use something like this. Be sure to check the `size()` of the string list and set the appropriate variables; i.e. don't set the `defusalMessagesFile` if there's file given for that parameter

```cpp
bz_APIStringList cmdLineParams;
cmdLineParams.tokenize(commandLine, ",");

// cmdLineParams->get(0) would be the death messages and index 1 would be defusal
```